.github/workflows/cherry-picks.yml

```diff
@@ -0,0 +1,41 @@
+name: Cherry Pick to remote repository
+on:
+  pull_request_target:
+    types: [closed, labeled]
+  issue_comment:
+    types: [created]
+jobs:
+  cross-repo-cherrypick:
+    name: Cherry pick to remote repository
+    runs-on: ubuntu-latest
+    # Only run when pull request is merged, or labeled
+    # or when a comment containing `/cherry-pick` is created
+    # and the author is a member, collaborator or owner
+    if: >
+      (
+        github.event_name == 'pull_request_target' &&
+        github.event.pull_request.merged
```

Comment:
Aren't we missing check for `labeled` ? I know that the action itself checks for labels but I'm wondering why you removed the check here?

Reply:
The check in the action I'm referring to:
https://github.com/jschmid1/cross-repo-cherrypick-action/blob/main/src/cherrypick.ts#L80

Reply:
it runs on _events_ that are either `merged` or `labeled` or `comment created`. This block prevents any the action from running when you add a label, but allows it when a comment is created.